src/main/java/seedu/exercise/ui/MainWindow.java

```diff
@@ -155,6 +152,27 @@ private void setChart() {
             chartPlaceholder.getChildren().add(new PieChartPanel(statistic).getRoot());
         }
     }
+
+    private void displayInitialList() {
+        changeTab(scheduleListTabPlaceholder);
+        infoDisplayPanel.showDefaultMessage();
+    }
+
+    private void initListenersForResourceListPanels() {
+        exerciseListPanel.setOnItemSelectListener(getListener());
+        regimeListPanel.setOnItemSelectListener(getListener());
+        scheduleListPanel.setOnItemSelectListener(getListener());
+    }
+
+    private ResourceListPanel.OnItemSelectListener getListener() {
+        return new ResourceListPanel.OnItemSelectListener() {
+            @Override
+            public void onItemSelect(Resource selected) {
+                infoDisplayPanel.update(selected);
+            }
+        };
+    }
+
```

Comment:
Please help to add documentation for this so that it is easier to follow. Thanks!

Reply:
will add in next update. thanks